gatsby-config.js

```diff
@@ -200,6 +200,8 @@ module.exports = {
         checkSupportedExtensions: false, // suppress warning about childImageSharp being null
       },
     },
+    "gatsby-plugin-image",
+    "gatsby-transformer-sharp", // Needed for dynamic images
```

Comment:
Considering we have `gatsby-transformer-sharp` declared at line: 198 under `gatsby-plugin-sharp` I doubt if this is needed?
You can simply move, the `gatsby-plugin-image` declaration above `gatsby-plugin-sharp` to line: 196, and hopefully that should work.

Reply:
Okay, Will Fix This 👀

Reply:
@Nikhil-Ladha Resolved this 💯